CONTRIBUTING.md

```diff
@@ -124,9 +124,38 @@ The maintenance branch's HEAD will be the target for our release tag:
 $ git tag -a -m "Release X.Y.Z" X.Y.Z
 ```
 
-### Push tags
+### Update branch-alias for dev-master (major and minor releases only)
```

Comment:
@tanlisu: See [branch-alias](https://getcomposer.org/doc/articles/aliases.md#branch-alias) for additional context.